src/gui/README.md

```diff
@@ -345,6 +345,19 @@ To remove all the rulers:
 gui::clear_rulers
 ```
 
+### Heat Maps
+
+To control the settings in the heat maps:
+
+```
+gui::set_heatmap name option value
+```
+
+Options description: 
+- ``name`` is the name of the heatmap. ``Power``, ``Routing`` or ``Placement``
+- ``option`` is the name of the option to modify. If option is ``rebuild`` the map will be destroyed and rebuilt.
+- ``value`` is the new value for the specified option
```

Comment:
This is a bit clunky - you always have to give a number for value even with rebuild (no arg needed) or ShowLengend (boolean).

Reply:
I've removed the need for the 3rd argument by making it 0 by default. The boolean option for ShowLegend is still needed. At the moment I don't know the type of the option until the option is found in the settings, so having tcl commands for those might be difficult to manage.